src/components/Popup.jsx

```diff
@@ -7,25 +7,25 @@ class Popup extends React.Component {
     this.popupBody = null;
     this.close = this.close.bind(this);
   }
-  
+
   render() {
     return (
       <div className="popup" ref={(c)=>{this.popupBody=c}} onClick={(e) => { return e.target === this.popupBody && this.close(e); }}>
         <div className="popup-title">
-          <button className="fa fa-close" onClick={this.close}></button>
+          <i className="fa fa-close" onClick={this.close}></i>
```

Comment:
Yes! Good catch. I was wondering why the linter didn't pick this up. I mainly did this to quickly get rid of some of the pesky default button styling going on, but I can find a workaround to keep the `onClick` handler attached to a button.